src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/CollectionModelBinder.cs

```diff
@@ -138,8 +141,8 @@ protected object CreateInstance(Type targetType)
         // Used when the ValueProvider contains the collection to be bound as a single element, e.g. the raw value
         // is [ "1", "2" ] and needs to be converted to an int[].
         // Internal for testing.
-        internal async Task<CollectionResult> BindSimpleCollection(
-            ModelBindingContext bindingContext,
+        public async Task<CollectionResult> BindSimpleCollection(
```

Comment:
I believe we have standardized on avoiding internal/internalsvisibleto - i'll get clarification on this one.